MovieDTT/MovieDTT/Views/HomePage.xaml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<TabbedPage xmlns="http://xamarin.com/schemas/2014/forms"
+            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+            xmlns:prism="clr-namespace:Prism.Mvvm;assembly=Prism.Forms"
+            xmlns:views="clr-namespace:MovieDTT.Views"
+			xmlns:appInfo="clr-namespace:MovieDTT.assembly=MovieDTT"
+            prism:ViewModelLocator.AutowireViewModel="True"
+			Title="Movie List"
+			x:Class="MovieDTT.Views.HomePage">
+	<!--<TabbedPage.ToolbarItems>
```

Comment:
commented out code, please remove it if not used.

Reply:
It was fixed